src/en/space-station-14/round-flow/proposals/station-ai.md

```diff
@@ -0,0 +1,162 @@
+  - Antagonist AI, “Malf”, should have an option or be able to spend some resources to completely lock the crew out. Will require more detailed balancing later.
+  - For “Malf”, the timeout should be reduced or bypassable. Again, might need some balance tweaking whether it’s just a pop up “you sure?” or a resource sink.
+- Borgs! Useful and somewhat plentiful, perfect for an emergency ~~snack~~ method of survival when the station’s powered off and the AI SMES is about to drain, with a downside of rendering the borg unable to synthesize voice (speak) when AI’s along for the ride, except for beeps and emotes. Does not require borg sync, but requires approval from the borg even if synced. Malf AI can bypass the approval.
+
+The AI satellite has several of the AI machines built into it, providing the base necessary resources for the AI, and some extra to make it actually useful to the crew.
+
+**Note:** Something noted by other people is how this system would interact with multiple AI. Two AI might be possible with either significant crowding and cut down resources, or a very annoyed crew as two AI fight one another for scraps. They should be able to lock one another out from a device, and require significant investment of power to pull control away. Of course, the crew can always just shoo them away with same near-instant action.
+
+## Vision - Cameras and Map
+
+The AI is not a biological entity, and it’s only barely a “physical” one. It doesn’t need to care about specifics of how things look, but rather whether or not an area is intact, safe, and secure. As such, the vision system is an optional ability that an AI player can shut down, whether to free space up for something else, or as an emergency measure.
+
+### Map Vision
+
+The “map” view replaces the entirety of the AI player's vision if the camera ability is shut down, or if there’s no cameras *and* they activate a composite vision ability. It should provide information about the station’s hull - walls, floors (or a glaring lack of one or being stripped down to a lattice), windows. It also should display machines that have an AI control wire - deemphasizing them if the wire’s been snipped, and potentially not showing them at all if there’s no camera that can see the machine (to make camera snipping actually work around APCs, air alarms). Fire alarms and atmosphere alarm status should potentially be shown over the area too - perhaps as an ability.
+
+While using the AI map vision, the crew can be seen via suit sensors, only if they have tracking on. Otherwise AI is unable to tell if someone is in the area without using cameras. A potential ability idea is to give cameras a motion detection cone that gets rendered for AI, and highlight entities within it - differently from tracked crew that enters it.
```

Comment:
Would be good and tie into motion alarm alerts. For stuff like feral slimes, spiders, crew with a cryptographic sequencer in Captain's Quarters, and crew in blood-red hardsuits holding C-20Rs in the Armory.